### DISCUSSION TITLE
Stable 3.0.0 version.

### DISCUSSION THREAD

**Discussion by inikulshin** [EasyNetQ/EasyNetQ.Management.Client#315] [General]
Can we publish a new stable version (not a pre-release, without suffix)?
We have issues creating our release packages with dependencies on pre-release packages (EasyNetQ.Management.Client.3.0.0-alpha7).

What exactly prevents publishing 3.0.0? There are no specific backlog items to implement...

**Comment by inikulshin** [comment]
@zidad @Pliner @micdenny @luigiberrettini

**Reply by zidad** [reply]
Sure, I'd say the only thing blocking is creating the release notes

**Reply by inikulshin** [reply]
It looks completely auto-generated:
https://github.com/EasyNetQ/EasyNetQ.Management.Client/releases

**Reply by inikulshin** [reply]
@zidad, @Pliner let's write all the "quick win" breaking changes (that require incrementing major version) we want to make (I'll make) before releasing `3.0.0`.

https://github.com/EasyNetQ/EasyNetQ.Management.Client/issues/300 ?
https://github.com/EasyNetQ/EasyNetQ.Management.Client/discussions/284 ?

Also, I suggest to extract `ExchangeName` (and maybe some more) similarly to `QueueName`.
And generate extensions with `QueueName` instead in `Queue`, `ExchangeName` instead of  `Exchange` and so on.

**Reply by zidad** [reply]
fine by me, @Pliner how are the release notes generated, manually or automatically?
